contents/docs/libraries/js/index.mdx

```diff
@@ -380,6 +380,13 @@ posthog.startSessionRecording()
 // Turns session recording off
 posthog.stopSessionRecording()
 
+// if you use ingestion controls they still apply,
+// you can pass override config to make sure the session starts
+// even if your ingestion controls would not have allowed it
+// so you could be sampling only 80% of sessions that match a particular linked flag
+// and still start recording for the current session using
+posthog.startSessionRecording({ linked_flag: true, sampling: true })
```

Comment:
1. Instead of a paragraph of comments, we can just do a paragraph.
2. It still is a bit confusing from this what the overrides do.

I'll take a crack at fixing this, think we need to adjust some of the surrounding information anyways.